Formula/googletest.rb

```diff
@@ -4,6 +4,7 @@ class Googletest < Formula
   url "https://github.com/google/googletest/archive/release-1.11.0.tar.gz"
   sha256 "b4870bf121ff7795ba20d20bcdd8627b8e088f2d1dab299a031c1034eddc93d5"
   license "BSD-3-Clause"
+  revision 1
```

Comment:
Nothing really changed here, so I think we can skip the `revision` bump. We may want to remove the `conflicts_with` lines in a separate syntax-only PR to avoid discarding the `:mojave` bottle for this formula.

Reply:
I've redacted this change in my latest force-push and will make a separate pull request as you have suggested